erlang/blocktree/blocktree.erl

```diff
@@ -241,7 +241,12 @@ generate_new_block(PreviousBlockId, VerifierData)
     #block{previous_id = PreviousBlockId, height = Height, transactions = BlockTransactions}.
     
 
-    
-    
-    
-    
+get_block_by_id(TreeData, Id)    
+  when is_record(TreeData, tree_data) ->
```

Comment:
not related to this code, but what does this function return when "when ...." is false?

Reply:
DDD: